pkg/ocm/core.go

```diff
@@ -112,7 +112,8 @@ func (o *OCMInstance) ListManagedClusters(clusters []*v1.Cluster) []*v1.Cluster
 	var clusterList []*v1.Cluster
 
 	for _, cluster := range clusters {
-		if cluster.Product().ID() != "ocp" {
+	    cluster_id := cluster.Product().ID()
+		if cluster_id != "ocp" && cluster_id != "aro" {
```

Comment:
```diff
diff --git a/pkg/ocm/core.go b/pkg/ocm/core.go
index c826159..69eecf5 100644
--- a/pkg/ocm/core.go
+++ b/pkg/ocm/core.go
@@ -5,6 +5,7 @@ import (
        "fmt"
        "os"
        "regexp"
+       "slices"
 
        sdk "github.com/openshift-online/ocm-sdk-go"
        v1 "github.com/openshift-online/ocm-sdk-go/clustersmgmt/v1"
@@ -112,8 +113,7 @@ func (o *OCMInstance) ListManagedClusters(clusters []*v1.Cluster) []*v1.Cluster
        var clusterList []*v1.Cluster
 
        for _, cluster := range clusters {
-           cluster_id := cluster.Product().ID()
-               if cluster_id != "ocp" && cluster_id != "aro" {
+               if !slices.Contains([]string{"ocp", "aro"}, cluster.Product().ID()) {
                        clusterList = append(clusterList, cluster)
                }
        }
```

Lets use the slices package to check whether the cluster id is in the slice to perform the filtering.

Reply:
done